lib/ProductOpener/Lang.pm

```diff
@@ -388,11 +388,11 @@ PO
 
 # Load stored %Lang from Lang.sto
 
-my $path = "$data_root/Lang.${server_domain}.sto";
+my $path = "$data_root/data/Lang.${server_domain}.sto";
```

Comment:
this might make production deployment fail (not sure how and how often build_lang is run there)

Reply:
Previously this was run at each start of the container. Now it's run as image is built. It only depends on files in the git, so this is the right time to compute it.

Reply:
I was mentioning production because the deployment there is not automated, so the file is probably located in `$data_root/Lang.${server_domain}.sto` and I'm not sure when `build_lang.pl` is ran in production (maybe a cronjob ?)

Reply:
as said in conf call, @stephanegigandet will run build_lang.pl on prod as he deploy (this is a necessary step), which will generate file in the right place. It might then remove the old one.